en-us/config.md

```diff
@@ -128,37 +128,37 @@ metrics = true
-- `data_path`: 链数据所在目录
+- `privkey`: Private key of the node, also the only identifier of the node, used when bootstraps the node. In order for other nodes to connect, it needs to expose address and corresponding public key. If it's a block producing node, its address needs to be included in consensus `verifier_list`
+- `data_path`: where chain data is stored
 - `graphql`:
-  - `listening_address`: GraphQL 监听地址
-  - `graphql_uri`: GraphQL 服务访问路径
-  - `graphiql_uri`: GraphiQL 访问路径
-  - `workers`: 处理 http 的线程数量，填 0 的话，会默认按 CPU 的核数
-  - `maxconn`: 最大连接数
+  - `listening_address`: listening address of GraphQL
+  - `graphql_uri`: URL to access GraphQL service
+  - `graphiql_uri`: URL to access GraphiQL
+  - `workers`: number of thread to handle http request. If 0 is entered, will default use number of CPU core
+  - `maxconn`: max number of connection
 - `network`:
-  - `listening_address`: 链 p2p 网络监听地址
-  - `rpc_timeout`: RPC 调用（例如从其它节点拉交易）超时时间，单位为秒
-- `network.bootstraps`: 起链时连接的初始节点信息
-  - `pubkey`: 公钥
-  - `address`: 网络地址
-- `mempool`: 交易池相关配置
-  - `pool_size`: 交易池大小
-  - `broadcast_txs_size`: 一次批量广播的交易数量
-  - `broadcast_txs_interval`: 每次广播交易的时间间隔，单位 ms
+  - `listening_address`: listening address of the chain
+  - `rpc_timeout`: timeout of RPC call (Ex: pull transaction from other nodes), measured in second
+- `network.bootstraps`: starting node info when bootstraps the chain
+  - `pubkey`: public key
+  - `address`: network address
+- `mempool`: exchange pool configurations
+  - `pool_size`: size of pool
+  - `broadcast_txs_size`: number of transaction in one broadcast
+  - `broadcast_txs_interval`: interval between each broadcast, measured in millisecond
 - `executor`:
-  - `light`: 设为 true 时，节点将只保存最新高度的 state
-- `logger`: 日志相关配置
-  - `filter`: 全局日志级别
-  - `log_to_console`: 是否输出日志到 console，生产环境建议设为 false
-  - `console_show_file_and_line`: 当 `log_to_console` 和本配置都置为 true 时，console 输出的日志里会包含日志打印处的文件和行数。本地通过日志调试时有用，一般可以设为 false。
-  - `log_to_file`: 是否输出日志到文件
-  - `metrics`: 是否输出 metrics。logger 模块中有专门的 metrics 输出函数，如有需要，可以用来输出 metrics 日志，不受全局日志级别的影响，且对应的日志会输出到专门的文件。
-  - `log_path`: 会在该路径生成两个日志文件：`muta.log` 和 `metrics.log`。`metrics.log`中包含了专门的 metrics 日志，`muta.log` 中包含了其它所有 log 输出。
-
-## 日志示例
-
-文件中的日志均为 json 格式，方便用程序处理。其中 message 一般为一个嵌套的 json 结构，用来表达结构化信息。
+  - `light`: when set to true, node will only keep the state of latest block hight
+- `logger`: logger configuration
+  - `filter`: log level globally
+  - `log_to_console`: whether output the log to console, it's recommended to set to false in production
+  - `console_show_file_and_line`: when `log_to_console` and `console_show_file_and_line` are both true, log will include file and number of lines
+  - `log_to_file`: whether output log to file
+  - `metrics`: whether output metrics, there are metric functions in logger module. It's independent from global log level, and it will output to specific files
```

Comment:
whether output metrics. There are metric functions which is independent from global log level in logger module, and if it's needed, it can be used to output metrics to specific files.